src/sqlfluff/core/rules/base.py

```diff
@@ -130,17 +130,17 @@ def is_trivial(self):
 
     def __repr__(self):
         if self.edit_type == "delete":
-            detail = "del:{0!r}".format(self.anchor.raw)
+            detail = "delete:{0!r}".format(self.anchor.raw)
         elif self.edit_type in ("edit", "create"):
             if hasattr(self.edit, "raw"):
                 new_detail = self.edit.raw
             else:
                 new_detail = "".join(s.raw for s in self.edit)
 
             if self.edit_type == "edit":
-                detail = "edt:{0!r}>{1!r}".format(self.anchor.raw, new_detail)
+                detail = "edit:{0!r}>{1!r}".format(self.anchor.raw, new_detail)
             else:
-                detail = "crt:{0!r}".format(new_detail)
+                detail = "create:{0!r}".format(new_detail)

```

Comment:
Codecov failing here. I added these changes as I had to look at the code to understand what crt meant